lib/src/main/java/at/bitfire/cert4android/UserDecisionUi.kt

```diff
@@ -0,0 +1,118 @@
+package at.bitfire.cert4android
+
+import android.Manifest
+import android.annotation.SuppressLint
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import android.content.pm.PackageManager
+import androidx.core.app.ActivityCompat
+import androidx.core.app.NotificationCompat
+import kotlinx.coroutines.suspendCancellableCoroutine
+import java.security.cert.X509Certificate
+import kotlin.coroutines.Continuation
+import kotlin.coroutines.resume
+
+class UserDecisionUi private constructor(
+    val context: Context
```

Comment:
I prefer passing context every time for `check` and `onUserDecision` so it doesn't get leaked. But I guess it shouldn't be a problem.

Reply:
It only stores the applicationContext. We only use it for notifications so it should be fine.